### ISSUE TITLE
Image issue after upgrade to 2.4

### ISSUE THREAD

**Issue by mwestergaard** [veda-consulting/uk.co.vedaconsulting.mosaico#384]
I upgraded from version 2.3.1572289217 to 2.4.1592820024 and now have what appears to be the issue described by @twomice in https://github.com/veda-consulting-company/uk.co.vedaconsulting.mosaico/issues/373. I use gd but get the same result with imagick. I verified that this patch is present: https://github.com/deepak-srivastava/uk.co.vedaconsulting.mosaico/commit/c535f438f05ee450c8798bd692c6add77965c8c0. I am on Wordpress 5.4.2 and CiviCRM 5.26.2. I tested in "troubleshooting mode" (all plugins, except CiviCRM, disabled and switched to twenty-twenty theme). I used the stock versafix-1 template. This is my dev system. I have a nearly identical production system that is still on 2.3.1572289217 and it is fine.

**Comment by themak1985** [comment]
Here to share that I have the same issue - it's like #373 #340 again.

**Comment by kaldonia** [comment]
I have exact the same issue, also with Wordpress 5.4.2 and CiviCRM 5.26.2, Mosaico 2.4.1592820024

```
$> php -r '$i=new Imagick();print_r($i->getVersion());'                
Array
(
    [versionNumber] => 1690
    [versionString] => ImageMagick 6.9.10-23 Q16 x86_64 20190101 https://imagemagick.org
)
```
```
$> php --version                                       
PHP 7.3.14-1~deb10u1 (cli) (built: Feb 16 2020 15:07:23) ( NTS )
Copyright (c) 1997-2018 The PHP Group
Zend Engine v3.3.14, Copyright (c) 1998-2018 Zend Technologies
    with Zend OPcache v7.3.14-1~deb10u1, Copyright (c) 1999-2018, by Zend Technologies
```
```
$> uname -a
Linux v220200232938109001 4.19.0-9-amd64 #1 SMP Debian 4.19.118-2 (2020-04-29) x86_64 GNU/Linux
```
```
$> dpkg -s imagemagick   
Version: 8:6.9.10.23+dfsg-2.1

```
I am willing to support in the issue, but don't know where to start. Any ideas? How can I help?

**Comment by mwestergaard** [comment]
I found that the image issue cleared up after switching the Mosaico graphics driver (Administer -> CiviMail -> Mosaico Settings) to "(Deprecated) Direct ImageMagick API". I have imagemagick 8:6.9.2.10, but was originally using gd without issue.

**Comment by mattwire** [comment]
Fixed in 2.5
